modulefiles/mpi/compilerName/compilerVersion/mpiName/mpiVersion/met/met.lua

```diff
@@ -0,0 +1,34 @@
+help([[
```

Comment:
Need one in the `compiler` hierarchy as well, in case we build a serial version.